lib/phlex/sgml.rb

```diff
@@ -406,8 +406,14 @@ def __build_attributes__(attributes, buffer:)
 							end
 						}, buffer: buffer
 					)
+				when Array
+					if v.all? { |el| el.is_a?(String) || el.is_a?(Symbol) }
+						buffer << " " << name << '="' << ERB::Escape.html_escape(v.join(" ")) << '"'
+					else
+						raise ArgumentError, "Element attributes must be either a Boolean, a String, a Symbol, an Array of Strings or Symbols, or a Hash with values of one of these types"
+					end
```

Comment:
I think we could probably skip the type check here and just `join`.

```suggestion
					buffer << " " << name << '="' << ERB::Escape.html_escape(v.join(" ")) << '"'
```